docs/osquery/osquery.asciidoc

```diff
@@ -3,16 +3,16 @@
 [[osquery]]
 = Osquery
 
-https://osquery.io[Osquery] is an open source tool that lets you query operating systems, like a database, providing you with visibility into your infrastructure and operating systems.
+https://osquery.io[Osquery] is an open source tool that lets you query operating systems like a database, providing you with visibility into your infrastructure and operating systems.
 Using basic SQL commands, you can ask questions about devices, such as servers, 
 Docker containers, and computers running Linux, macOS, or Windows.
 The https://osquery.io/schema[extensive schema] helps with a variety of use cases,
 including vulnerability detection, compliance monitoring, incident investigations, and more.  
 
-With Osquery, you can:
+With Osquery in {kib}, you can:
 
  * Run live queries for one or more agents
- * Schedule queries to capture changes to OS state over time
+ * Schedule query packs to capture changes to OS state over time
  * View a history of past queries and their results
  * Save queries and build a library of queries for specific use cases
```

Comment:
I think it's an important thing to note somewhere, because we've received some questions about it. But I definitely see your point. I'll try to find a better spot for it.

Reply:
I ended up moving it to the "Osquery results" section - seems like a better fit there.